raiden/utils/upgrades.py

```diff
@@ -55,9 +58,9 @@ def get_db_version(db_filename: Path):
         return 0
 
 
-def _run_upgrade_func(cursor: sqlite3.Cursor, func: Callable, version: int) -> int:
+def _run_upgrade_func(cursor: sqlite3.Cursor, func: Callable, version: int, **args) -> int:
```

Comment:
convention is to name the argument as `kwargs` to indicate this is a dictionary rather than a list. `args` usually refers to this argument being a list. Though in your case, the `**` kinda does that but i would also rename this var

Reply:
sure. You remember that this was a list before, this is why the name is still args.